<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autonimo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../autonimo.qrc">
    <normaloff>:/img/table.png</normaloff>:/img/table.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_tasks">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Tasks</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QTreeView" name="treeView_tasks_available"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_task"/>
   <addaction name="action_save_task"/>
   <addaction name="action_save_task_as"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_task"/>
    <addaction name="action_save_task"/>
    <addaction name="action_save_task_as"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Components</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeView" name="treeView_comps_available"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_open_task">
   <property name="icon">
    <iconset resource="../autonimo.qrc">
     <normaloff>:/img/folder-horizontal-open.png</normaloff>:/img/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Open task...</string>
   </property>
   <property name="toolTip">
    <string>Open task</string>
   </property>
  </action>
  <action name="action_save_task">
   <property name="icon">
    <iconset resource="../autonimo.qrc">
     <normaloff>:/img/disk.png</normaloff>:/img/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save task</string>
   </property>
   <property name="toolTip">
    <string>Save task</string>
   </property>
  </action>
  <action name="action_save_task_as">
   <property name="icon">
    <iconset resource="../autonimo.qrc">
     <normaloff>:/img/disk-rename.png</normaloff>:/img/disk-rename.png</iconset>
   </property>
   <property name="text">
    <string>Save task as...</string>
   </property>
   <property name="toolTip">
    <string>Save task as</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../autonimo.qrc">
     <normaloff>:/img/balloon-smiley.png</normaloff>:/img/balloon-smiley.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../autonimo.qrc"/>
 </resources>
 <connections/>
</ui>
